src/bin/hal/cmd/tx.rs

```diff
@@ -116,10 +116,10 @@ fn create_input(input: InputInfo) -> TxIn {
 	TxIn {
 		previous_output: outpoint_from_input_info(&input),
 		script_sig: input.script_sig.map(create_script_sig).unwrap_or_default(),
-		sequence: input.sequence.unwrap_or_default(),
+		sequence: bitcoin::Sequence::from_consensus(input.sequence.unwrap_or_default()),
```

Comment:
Same here as locktime.

Reply:
Do you mean `from_height`?

Reply:
yeah I think so. something that does validation instead of assuming consensus did validation